### PR TITLE
FIX: Let modified_pulses_only can be changed

### PR DESCRIPTION
**Pull request by qux-bbb**
The original logic will cause modified_pulses_only is always False.

### CODE REVIEW

**Comment by kamil-certat** [comment]
Hey, thanks for catching that! Could you please add an entry to the Changelog.md with your fix? A test would be also great, but I think it had to be about the whole feature, so I believe we can go without it eventually.

**Comment by aaronkaplan** [comment]
reviewed. Clear logic bug, thx for catching it @qux-bbb . Pulling that in.

**Comment by aaronkaplan** [comment]
@sebix could you please also double check? It's trivial . thx. if all ok, pls press "merge" :) thx

File: CHANGELOG.md [hunk 1]
```diff
@@ -61,8 +61,9 @@
   - Remove undocumented and unused attributes of `StompCollectorBot` instances:
     `ssl_ca_cert`, `ssl_cl_cert`, `ssl_cl_cert_key`.
   - Minor fixes/improvements and some refactoring (see also above: *Core*...).
-- `intelmq.bots.collectors.shadowserver.collector_reports_api`:
-  - The 'json' option is no longer supported as the 'csv' option provides better performance (PR#2372 by elsif2).
+- `intelmq.bots.collectors.alienvault_otx.collector` (PR#2449 by qux-bbb):
```
Comment (kamil-certat) [inline CHANGELOG.md:67]: It looks like there is a merging issue removing previous changelog entry. Could you please restore it?

**Review by kamil-certat — APPROVED** [review]

**Pull request merged**